.circleci/config.yml

```diff
@@ -1,38 +1,42 @@
 common-job-attribues: &common-job-attributes
   docker:
     - image: circleci/golang:1.17
-
 install-aws-cli: &install-aws-cli
   run:
     name: Install AWS CLI
     command: |
       sudo apt-get install -y python3-pip
       sudo pip3 install --upgrade awscli
-
 version: 2.1
 jobs:
   master:
-    <<: *common-job-attributes
+    !!merge <<: *common-job-attributes
```

Comment:
Looks a bit wonky. Also, I don't think this project is configured to use our CircleCI. We forked a public repo with its own CircleCI config.